composer.json

```diff
@@ -27,7 +27,7 @@
         "symfony/http-foundation": "~2.4|~3.0|^4.0",
         "symfony/phpunit-bridge": "~3.3|^4.0",
         "symfony/routing": "~2.3|~3.0|^4.0",
-        "twig/twig":       "~1.16|~2.0"
+        "twig/twig":       "~1.38|~2.0"
```

Comment:
lowest 2.x version having that filter is 2.7

Reply:
Well, I tried to increase required Twig version and everything is broken... :disappointed_relieved: Not sure of the reason, tho

Reply:
I see that it was failing even before. Resolving this part for now...